.openshift-ci/gcp/command.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# The script runs a command with the kernel-packer repo as current working
+# directory.
+
+set -e
+
+runCommand() {
+    local GCP_VM_NAME="$1"
+    shift
+    local GCP_VM_COMMAND="$1"
+    shift
+
+    [ -z "$GCP_VM_NAME" ] && \
+        echo "error: missing parameter GCP_VM_NAME" && return 1
+
+    [ -z "$GCP_VM_COMMAND" ] && \
+        echo "error: missing parameter GCP_VM_COMMAND" && return 1
```

Comment:
I see this pattern in some other scripts, so if you do decide to implement the `die` function it'd be nice to use it across all of them (super annoying, I know)

Reply:
Makes sense, I've added the function. Since there is no common script they import, I leave some duplication in there. I would love to address this and retry function as a refactoring for the next steps, what do you think?